sources/packages/backend/apps/api/src/route-controllers/student/student.controller.service.ts

```diff
@@ -154,9 +154,11 @@ export class StudentControllerService {
       );
     }
 
+    const encodedFileName = encodeURIComponent(studentFile.fileName);
+
     response.setHeader(
       "Content-Disposition",
-      `attachment; filename=${studentFile.fileName}`,
+      `attachment; filename*=UTF-8''${encodedFileName}`,
```

Comment:
Great finding. 
I see that it follows the `Extended notation` example from [RFC5987](https://www.rfc-editor.org/rfc/rfc5987#section-3.2.2)

I believe the `encodeURIComponent()` sufficient not required to do further replace as stated here. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent#examples 
![image](https://github.com/user-attachments/assets/d8f614b5-17bb-4de1-a6d6-0f971f5bf664)


@andrewsignori-aot @andrepestana-aot @guru-aot @lewischen-aot

Reply:
Agreed @dheepak-aot

Reply:
Looks good.